tls/dub.sdl

```diff
@@ -16,6 +16,8 @@ configuration "openssl-mscoff" {
 	platforms "windows-x86_mscoff" "windows-x86_64" "windows-x86-ldc"
 	dependency "openssl" version="~>3.1"
 	subConfiguration "openssl" "library-manual-version"
+	dependency "openssl-static" version="~>1.0"
```

Comment:
New dependency ? Why ?

Reply:
That's just to allow forcing the "none" configuration. I'll try to rework that into an optional dependency to avoid the download.